doc/data-structures.rst

```diff
@@ -401,6 +401,20 @@ operations keep around coordinates:
     list(ds[['x']])
```

Comment:
Not sure why there is a `list` here either :) but @jbusecke I think you just need to add `ds[['x', 'temperature']]` as an example here since the text already talks about passing a list of variable names.

Reply:
Oh I see. Honestly I find the `ds[['x']]` line quite confusing without context. Why should I chose one of the dimensions?
Should I remove the `list(...)` calls alltoghether?

Reply:
I think the `list` calls need to be removed.

And I think changing `ds[['x']]` to `ds[['x', 'temperature']]` is probably all that is needed.

Reply:
Oh the `.get` example is great too, let's keep that.